_layouts/sidenav.html

```diff
@@ -16,9 +16,16 @@
     <ul class="usa-sidenav">
       <li class="usa-sidenav__item">
         <!-- TODO: Parent item should not be clickable -->
```

Comment:
I think we're fixing the problem likely being described in this comment? Should we remove it?

```suggestion
```